cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -192,6 +195,52 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
       optionalResultExplanation = "Note that it can only be created once and that the index name must be unique.",
       assertions = _ => assertTokenLookupIndexExists("rel_type_lookup_index", isNodeIndex = false)
     )
+    prepareAndTestQuery(
+      title = "Create a token lookup index specifying the index provider",
+      text = "Token lookup indexes allow setting the index provider using the `OPTIONS` clause. " +
+        "Note that the index is not immediately available, but is created in the background.",
+      prepare = _ => executePreparationQueries(List("DROP INDEX node_label_lookup_index IF EXISTS")),
+      queryText = s"CREATE LOOKUP INDEX node_label_lookup_index_2 FOR (n) ON EACH labels(n) OPTIONS {indexProvider: '${TokenIndexProvider.DESCRIPTOR.name()}'}",
+      optionalResultExplanation = "Currently only one provider is available for token lookup indexes.",
```

Comment:
I think it would be nice to name the provider in this sentence, like you did for text indexes.